### DISCUSSION TITLE
Welcome to dragonwell8 Discussions!

### DISCUSSION THREAD

**Discussion by j0826** [alibaba/dragonwell8#220] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by luchsh** [comment]
You guys all know me well enough :);

Is this thread official? or just for testing? anyway, is there anybody watching this thread

days_since_initial_post = 2
